* looking for rhythmbox-devel@gnome.org--2004/rhythmbox--main--0.8--patch-133 to compare with
* comparing to rhythmbox-devel@gnome.org--2004/rhythmbox--main--0.8--patch-133
M  sources/rb-ipod-source.c

* modified files

--- orig/sources/rb-ipod-source.c
+++ mod/sources/rb-ipod-source.c
@@ -236,11 +236,11 @@
 		/* Set disc number */
 		if (song->cd_nr != 0) {
 			GValue value = {0, };
-			g_value_init (&value, G_TYPE_ULONG);
-			g_value_set_ulong (&value, song->cd_nr);
-			rhythmdb_entry_set (RHYTHMDB (ctxt->db), entry, 
-					    RHYTHMDB_PROP_DISC_NUMBER, 
-					    &value);
+			g_value_init (&value, G_TYPE_INT);
+			g_value_set_int (&value, song->cd_nr);
+			entry_set_locked (RHYTHMDB (ctxt->db), entry, 
+					  RHYTHMDB_PROP_DISC_NUMBER, 
+					  &value);
 			g_value_unset (&value);
 		}
 		



